<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tj_compiler</class>
 <widget class="QMainWindow" name="tj_compiler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TJ_Compiler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextBrowser" name="fileDisplayBrowser"/>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fileInput">
        <property name="text">
         <string>选择代码文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="parserfileInput">
        <property name="text">
         <string>选取语法文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lexicalBtn">
        <property name="text">
         <string>词法分析</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="parserBtn">
        <property name="text">
         <string>语法分析</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="allBtn">
        <property name="text">
         <string>一键完成</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
